R/functions_analysis.R

```diff
@@ -0,0 +1,55 @@
+#' Calculate enrichment of cells per sample per cluster.
```

Comment:
Very good, that file still needed some good documentation

Reply:
Do you mean, it still needs some good documentation? 

I'd like to do this once I tackle the documentation issue.

Reply:
No before the functions simply missed a documentation.